### ISSUE TITLE
Excess Inventory transfer loses enchantments and tempering

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#309]
Need to carry along _ExtraLists_ with the moved items. Should be possible a la _QuickLoot RE_

**Comment by SteveTownsend** [comment]
Initial implementation just never transfers [player-enahcnated ot termpered items. Test evidence:
```
2021-07-10 17:45:33.441    debug   1288 Player-enchanted Item Iron Mace/0x00013982, exempt from Excess Inventory
```

**Comment by SteveTownsend** [comment]
Fix applied for v3.9.4.2
